pl/lessons/libraries/benchee.md

```diff
@@ -0,0 +1,335 @@
+---
+version: 1.0.0
+title: Benchee
+---
+
+Nie możemy po prostu przypuszczać, które funkcje są szybkie a które są powolne - aby to ustalić potrzebujemy rzeczywistych pomiarów. Tu z pomoca przychodzi analiza porównacza. W tej lekcji nauczymy się jak łatwo jest zmierzyć szybkość naszego kodu.
+
+{% include toc.html %}
+
+# O Benchee
+
+Chociaż istnieje [funkcja w Erlangu](http://erlang.org/doc/man/timer.html#tc-1), która może być użyta do bardzo podstawowego pomiaru czasu wykonania funkcji, nie jest tak latwa w użytkowaniu jak niektóre z dostępnych narzędzi. Nie daje Ci wielu pomiarów, ktore sa niezbedne do prawodłowego przeprowadzenia statystyk, dlatego skorzystamy z [Benchee](https://github.com/PragTob/benchee). Benchee dostarcza nam wielu statystyk z łatwymi do porównania scenariuszami, wspaniałą cechą, która pozwala nam przetestować różne dane wejściowe na funkcjach które testujemy i kilka różnych formaterów, które możemy wykorzystać do wyświetlania naszych wyników.
+
+# Użytkowanie
+
+Aby dodać Benchee do projektu, umieść go jako zależność w pliku `mix.exs`:
+
+```elixir
+defp deps do
+  [{:benchee, "~> 0.9", only: :dev}]
+end
+```
+
+Następnie wykonujemy:
+
+```shell
+$ mix deps.get
+...
+$ mix compile
+```
+
+Pierwsze polecenie pobiera i instaluje Benchee. Możesz zostać poproszony o zainstalowanie Hex wraz z nim. Druga kompiluje aplikację Benchee. Teraz jesteśmy gotowi napisać nasz pierwszy test wydajości!
```

Comment:
Probably it should be `Drugie` instead of `Druga`, because we're talking about `polecenie` in this context.